server/apps/immich/src/config/profile-image-upload.config.ts

```diff
@@ -38,8 +38,7 @@ function destination(req: Request, file: Express.Multer.File, cb: any) {
 
   const user = req.user as AuthUserDto;
 
-  const basePath = APP_UPLOAD_LOCATION;
-  const profileImageLocation = `${basePath}/${user.id}/profile`;
+  const profileImageLocation = join(APP_UPLOAD_LOCATION, user.id, 'profile');
```

Comment:
Should this go in the media-cache path instead?

Reply:
I thought about that, but profile image is an uploaded image so that can't really be re-generated again. In my mind `/media-cache` should be able to be regenerated from '/media`, so I thought about leaving it there.